src/NotificationTarget.php

```diff
@@ -374,9 +425,9 @@ public static function getInstance($item, $event = '', $options = [])
     /**
      * Get the expected notification target class name for a given itemtype
      *
-     * @param string $itemtype
+     * @param class-string<CommonGLPI> $itemtype
      *
-     * @return string
+     * @return class-string<NotificationTarget> // @todo review, do the plugin returns a NotificationTarget instance
```

Comment:
Remove the TODO.

Yes, plugins are supposed to return a `NotificationTarget` class.